README.md

```diff
@@ -65,20 +65,66 @@ yarn audit
 
 ### Configuration
 
-You don't need to [Configure Gimbal](./packages/gimbal/docs/config), but we understand that defaults are _optimistic_, at least for existing projects that want to introduce performance budgeting. 
+You don't need to [Configure Gimbal](./packages/gimbal/docs/config), but we understand that defaults are _optimistic_, at least for existing projects that want to introduce performance budgeting.
 
-Here's a **sample `.gimbalrc.yml` config** file:
+To ease you be ready to use, let's start with some **sample `.gimbalrc.yml` config** files:
+
+1. Minimal
+2. Minimal with all native audits
+3. Using other audit plugins and more sample configurations
+
```

Comment:
I am proposing three levels of configuration files, from a minimal to a more complex sample configuration file. Actual configuration sample leads to some potential exceptions when executing for the first time and may lead one to quit when facing the exceptions.